lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -278,13 +278,19 @@ getTxWitnessTag = getWord8 >>= \case
 getMessage :: Get Message
 getMessage = label "getMessage" $ do
     size <- fromIntegral <$> getWord16be
+
+    -- We lazily compute the hash of the message (needed for e.g transactions)
```

Comment:
The hash of the message is called the fragment-id in Jormungandr's terminology, maybe good to refer to it as is?

Reply:
Had to call the variable `fragId` to avoid shadowing the `fragmentId`-function, unfortunately though